src/app/admin/users/PageUsers.tsx

```diff
@@ -27,6 +27,7 @@ import {
   Wrap,
   WrapItem,
 } from '@chakra-ui/react';
+import { useAutoAnimate } from '@formkit/auto-animate/react';
```

Comment:
It would be great if you could directly do it on the `DataList` component. Here is some code exemple from another project that use :rocket: Start UI [web] https://github.com/BearStudio/start-repo/blob/main/src/components/DataList/index.tsx#L240-L268